src/ansys/fluent/core/meta.py

```diff
@@ -738,3 +735,29 @@ def Create(self, name=None):
     @CommandArgs(Create, "name")
     def type(self):
         return "string"
+
+
+class MetaClassError(ValueError):
+    """Custom meta class errors."""
+
+    def __init__(self, error):
+        super().__init__(error)
+
+    @classmethod
+    def DISALLOWED_VALUES(
```

Comment:
Are we going to upgrade this `DISALLOWED_VALUES` to an exception class as well so that it can be caught directly (similarly to what was done for `FluentConnectionError.PortNotProvidedError` and the others)?

Reply:
We haven't updated it with latest change yet. We will do it soon.

Reply:
Done.